src/stores/reduce-store.ts

```diff
@@ -161,10 +175,10 @@ export abstract class ReduceStore<TState> extends FluxReduceStore<TState, Dispat
      * Check if action should handled.
      * By default always return true.
      *
-     * @param {Object} action - Action payload data.
-     * @param {TState} state - Updated store state.
+     * @param action Action payload data.
+     * @param state Updated store state.
      */
-    protected shouldHandleAction(action: Object, state: TState): boolean {
+    protected shouldHandleAction(action: DispatcherMessage, state: TState): boolean {

```

Comment:
Not it's getting an object not an action:
_action Samples_:
```
// SimplrAction
{type: "SIMPLR_ACTION", action: SomeKindOfAction}
// FluxAction
{type: "SOME_ACTION", id: "action-id-1"}
```
/cc @DovydasNavickas